### ISSUE TITLE
Develop sensible naming conventions consistent across interfaces

### ISSUE THREAD

**Issue by chrisgorgo** [nipy/nipype#5]
E.g. if it's a file, is it called xxxfile? Or just xxx?

This is currently being done haphazardly even within individual classes, such as Flirt in fsl.

I think we should do this early and do it once - it will cause serious breakage of existing pypelines.

http://sourceforge.net/apps/trac/nipy/ticket/48

**Comment by djarecka** [comment]
@chrisfilo - do you mean files from `InputSpec`/`OutputSpec`? Have you discussed and agreed on convention?

**Comment by chrisfilo** [comment]
Yes. Could be possible in the new API since lots of things will break anyway.
